mysite/settings.py

```diff
@@ -25,6 +25,7 @@
 
 MIDDLEWARE = [
     'django.middleware.security.SecurityMiddleware',
+    'whitenoise.middleware.WhiteNoiseMiddleware',
```

Comment:
Does the tutorial require you to use middleware in exactly this place? This suggests that you should modify the WSGI module: https://tutorial-extensions.djangogirls.org/en/heroku/.
WDYT?

Reply:
I changed wsgi. When trying to do commit I get an error pylint: mysite/wsgi.py:6:0: E0611: No name 'DjangoWhiteNoise' in module 'whitenoise.django' (no-name-in-module)